src/main/kotlin/com/chattriggers/ctjs/api/client/Sound.kt

```diff
@@ -260,6 +258,8 @@ class Sound(private val config: NativeObject) {
      */
     @JvmOverloads
     fun play(delay: Int = 0) {
+        require (World.isLoaded()) { "Can not play a custom sound outside the world" }
```

Comment:
Hmm, this is a surprising restriction from a user perspective. Let's at least mention it in the class's ksdoc along with a TODO

Reply:
Yeah, maybe in the future someone can look into getting sounds to work, but I couldn't so this is a band-aid fix. Will do.